tests/ckeditor/basepath.js

```diff
@@ -31,6 +32,20 @@ describe( 'basePath', function() {
 		window.CKEDITOR_BASEPATH = 'http://foo.com/ckeditor/';
 		expect( CKEDITOR._getBasePath() ).to.equal( 'http://foo.com/ckeditor/' );
 	} );
+
+	function testGetBasePathFromTag( url, expectedBasePath ) {
```

Comment:
Alright, I'm glad I can say it was just a filtering mechanism issue. Please update benderjs-mocha to 0.1.3 and it should work fine.